### DISCUSSION TITLE
Change path/folder from Windows Open/Save dialog box

### DISCUSSION THREAD

**Discussion by hervehobbes** [adirh3/Fluent-Search#965] [Ideas]
Hello,

There is a program called [Quick Access Popup](https://www.quickaccesspopup.com/), with a feature called "Quick Launch", that enables you, when you have a Windows system "Open" or "Save/Save As" dialog box opened, to call a popup to select a folder from a predefined list. 

When you tend/try to be mouseless, it's a time saver. 

To use it, I call Quick Access Popup's Quick Launch feature then I start to type a part of a folder's name and when I hit enter, the path of the opened dialog box is changed.

If you implement a such feature, there's a "trap", after the selected path is opened, the filename that was in the dialog box must me "pasted" again.

You can see a Youtube's video of this feature [here](https://www.youtube.com/watch?v=4ibBAh9_Bsc) and [here](https://www.youtube.com/watch?v=YxZFQtuQCYE).

**Comment by adirh3** [comment]
You can already do that using Fluent Search, there are multiple ways when a dialog is open -
1. Simply search for a folder or file and press Enter - you will see it suggest as `Open in dialog`
2. Use the Quick Menu feature using a mouse gesture (Settings -> Quick menu) and select a folder you have pinned
3. Use the `Context` feature in Settings -> Context (toggle advanced), and use the Paste hotkeys to change the dialog folder to the last copied folder

**Reply by hervehobbes** [reply]
it's just a shame that the cursor is not repositioned on the name entry area

**Comment by hervehobbes** [comment]
Ah YES !
Thank you very  much.
